src/foam/nanos/crunch/Capability.js

```diff
@@ -505,9 +488,9 @@ foam.CLASS({
             var cap = (Capability) capabilityDAO.find(capId);
             if ( cap == null || cap.getLifecycleState() != foam.nanos.auth.LifecycleState.ACTIVE ) continue;
 
-            UserCapabilityJunction prereqUcj = crunchService.getJunctionForSubject(x, capId, subject);
+            UserCapabilityJunction prereqUcj = crunchService.getJunction(x, capId);
```

Comment:
does this need to be `getJunction(x, capId, ucj.getSubject(x));` ?

Reply:
updated in https://github.com/kgrgreer/foam3/pull/1623/commits/65c8a43d6d0d1d3e6b9eb621b20942eb1b240dcd